### ISSUE TITLE
ncb2, not co-repressor

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#2707]
SPBC30D10.02

new  term (and for binding partner)

**Comment by ValWood** [comment]
these are GTFs

- [ ] ncb2	transcription corepressor Ncb2 (predicted) -> transcription corepressor activity (GO:0003714)

**Comment by ValWood** [comment]
Hi @colinlog  sorry to bother you, I know you are super busy. I can't remember where we discussed that this was not a co-repressor. Do you have a reference I can use, or can you point me to the transcription group discussion? I remember we talked about   it but I did not do it soon enough that I remembered  the critical information.

**Comment by colinlog** [comment]
Dear Val,
According to the classic literature summarized by Koster et al (PMID:25957681) this is NC2b. Together with NC2a, that protein is a 'negative regulator of the TBP GTF. This would make nc2b a 'negative regulator of GTF GO:0016251' but that GO term does not exist...  Importantly, it may not be a negatively-acting factor for TBP in vivo, in that it may rather be part of the cycle of RNA polymerase II initiation + RNA polymerase II promoter clearance. In fact that was the conclusion that we reached in April, we decided to annotate  nc2b as a GTF (GO:0016251).

Furthermore, It appears that this protein is a subunit of the ATAC complex in metzoa, which is considered a coregulator (GO:0003712, more specifically a coactivator (GO:0003713)).

The best woul dbe to annotatte it to GTF as it is clearly involved in TBP biochemistry and TBP is **the** epitomy of a GTF.
Colin
